tools/src/h5stat/h5stat.c

```diff
@@ -172,102 +172,19 @@ struct handler_t {
 static const char *s_opts = "Aa:Ddm:EFfhGgl:sSTO:Vw:H:";
```

Comment:
I doubt more options will be added, but these longer lists should be in alpha order by short letter, makes it easier to know which short character was used. Long term it would be nice to change the option parsing function.